src/casp/services/logging.py

```diff
@@ -92,12 +92,21 @@ def format(self, record):
                 global_log_fields["logging.googleapis.com/trace"] = trace.get_trace(self.project_id)
                 global_log_fields["logging.googleapis.com/spanId"] = trace.span_id
 
+            locator = {
+                "filename": record.filename,
+                "lineno": record.lineno,
+                "funcname": record.funcName,
```

Comment:
Is this actually the variable name python logging records use?  "funcName" doesn't seem pythonic

Reply:
It sure is...

Reply:
Yeah, it's weird that such a core module would have this one random different casing but yeah, double checked:
https://github.com/python/cpython/blob/e69d068ad0bd6a25434ea476a647b635da4d82bb/Doc/library/logging.rst#logrecord-attributes